src/gui/owncloudgui.cpp

```diff
@@ -1131,5 +1137,24 @@ void ownCloudGui::slotRemoveDestroyedShareDialogs()
     }
 }
 
+void ownCloudGui::slotAbout()
+{
+    QString title = tr("About %1").arg(Theme::instance()->appNameGUI());
+    QString about = Theme::instance()->about();
+    QMessageBox *msgBox = new QMessageBox(this->_settingsDialog);
+#ifdef Q_OS_MAC
+    // From Qt doc: "On macOS, the window title is ignored (as required by the macOS Guidelines)."
+    msgBox->setText(title);
+#else
+    msgBox->setWindowTitle(title);
+#endif
+    msgBox->setAttribute(Qt::WA_DeleteOnClose, true);
+    msgBox->setTextFormat(Qt::RichText);
+    msgBox->setTextInteractionFlags(Qt::TextBrowserInteraction);
+    msgBox->setInformativeText("<qt><style> p{padding-bottom:0; margin-bottom:0;padding-top:2; margin-top: 2; font-size:smaller;};</style>"+about+"</qt>");
```

Comment:
is the style  "cargo cult"?

I guess we don't need this if it is in its own dialog.